udata_front/theme/gouvfr/assets/js/pages/OrganizationPublishingForm/Step2DescribeOrganization.vue

```diff
@@ -341,7 +341,8 @@
     if (siret.length === 14) {
       axios.get(config.search_siren_url, {
         params: {
-          q: siret
+          q: siret,
+          mtm_campaign: "publier-data-gouv"
```

Comment:
Nicolas talked with Xavier and they don't need details by environment. We can still implement it if we want